examples/Bunch/include/simulation_defines/param/radiationObserver.param

```diff
@@ -65,7 +65,10 @@ namespace picongpu
       /* + picongpu::PI -> turn observation direction 180 degrees towards -y */
 
       /* compute observation unit vector */
-      return vector_64(sinf(theta), cosf(theta), 0.0);
+      picongpu::float_32 sinTheta;
+      picongpu::float_32 cosTheta;
+      math::sincos(precisionCast<picongpu::float_32>(theta), sinTheta, cosTheta);
```

Comment:
Just a **question**:
@psychocoderHPC what would happen if I avoid the `precisionCast` here. Would it run in `double` or `single` precision or would it not run at all?